css/style.css

```diff
@@ -75,6 +75,189 @@ a.logo:hover {
   margin-left: 22px;
 }
 
+.works-container {
+  align-items: center;
+  padding: 20px;
+  display: grid;
+  background-color: #fff;
+}
+
+.card {
+  background-color: #fff;
+  border-radius: 5px;
+}
+
+h1.card-title {
+  color: #172b4d;
+}
+
+.work-details {
+  display: flex;
+}
+
+.work-item {
+  font-weight: 500;
+  font-size: 13px;
+  padding: 5px;
+  color: #7a869a;
+  line-height: 16px;
+}
```

Comment:
- Kindly add some space between your card container  so that your design will stick more to the Figma design 

![image](https://user-images.githubusercontent.com/95297251/191939162-b02b928c-ca53-41fe-a6c6-aaa4faf2b5f4.png)

![image](https://user-images.githubusercontent.com/95297251/191939663-630ee0b2-1d2e-4f43-9a18-8aec233887ca.png)